pkg/cloud/openstack/deployer.go

```diff
@@ -85,3 +85,14 @@ func (d *DeploymentClient) GetKubeConfig(cluster *clusterv1.Cluster, master *clu
 	}
 	return strings.TrimSpace(parts[1]), nil
 }
+
+func execCommand(name string, args ...string) string {
+	cmdOut, err := exec.Command(name, args...).Output()
+	s := strings.Join(append([]string{name}, args...), " ")
+	if err != nil {
+		klog.Warningf("failed to execute command %q: %v", s, err)
+	} else {
+		klog.Infof("successfully executed command %q", s)
```

Comment:
Can you please update the util function in `cluster-api`? I think it is OK to use `Errorf` if the command failed, but you can add the `else` logic for successful command, comments? @hidekazuna